src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -22,4 +22,6 @@ databaseChangeLog:
   - include:
       file: /db/changelog/changes/11-create-books-categories-table.yml
   - include:
-      file: /db/changelog/changes/12-loaddata-books-categories.yml
+      file: /db/changelog/changes/13-create-shopping-carts-table.yml
+  - include:
+      file: /db/changelog/changes/14-create-cart-items-table.yml
```

Comment:
What happened to the 12th script?

Reply:
I probably merged it wrong somewhere, thanks for noticing